### ISSUE TITLE
Polyratfun infinite loop in Print statement

### ISSUE THREAD

**Issue by benruijl** [vermaseren/form#37]
The following simple example generates an infinite loop in the Print statement:

```
S ep;
CF rat;
PolyRatFun rat(expand,ep,6);
L F  = rat(ep,ep);
Print;
.end
```

and when multiplied, it freezes:

```
S ep;
CF rat;
PolyRatFun rat(expand,ep,6);
L F  =rat(1,1)*rat(ep,ep);

.end
```

**Comment by vermaseren** [comment]
Quite nice.
I’ll have a look.
Clearly got stuck in a loop…..

Jos

**Comment by tueda** [comment]
At least, this issue is fixed in 1fd92ca0166772498a063c12ed29bd1636c11a83.
